packages/client/hmi-client/src/services/project.ts

```diff
@@ -102,12 +102,11 @@ async function deleteAsset(projectId: string, assetsType: string, assetId) {
 async function getRelatedArticles(aProject: Project): Promise<XDDArticle[]> {
 	const resp = await getAssets(aProject.id);
 	try {
+		// Dec 6th 2022:
 		// TODO: Speak with XDD Team about broken doc: 5f6d0e20a58f1dfd52184931
 		// Grab the 2nd of publication for related results because grabbing the first provides a broken doc id: 5f6d0e20a58f1dfd52184931
-		const listOfRelatedArticles = await getRelatedDocuments(
-			String(resp?.[ProjectAssetTypes.PUBLICATIONS][1].xdd_uri),
-			'xdd-covid-19'
-		);
+		const docId = resp ? resp?.[ProjectAssetTypes.PUBLICATIONS][1].xdd_uri : '';
```

Comment:
Slightly less verbose way you can do conditional assignment: 
`
const docId = resp?.[ProjectAssetTypes.PUBLICATIONS][1].xdd_uri ?? '';`